docs/src/index.md

```diff
@@ -121,16 +121,30 @@ skate('my-component', {
   // - oldValue: The old value. If type === 'created', this will be undefined.
   attributes: {
     'my-attribute': {
+      // The element's default value if not already specified on the element.
+      // Can also be a function that returns a value.
```

Comment:
Maybe document what argument that function receives (the `target` from looking at code below)?